Update guides and config reference - Closes #1278 (#1305)
LiskHQ/lisk-docs: modules/ROOT/pages/configuration/source.adoc

:url_management_pm2: management/source.adoc
:url_references_config: references/configuration.adoc

== Default configuration

> The {url_pm2_ecosystem_file}[PM2 ecosystem file^] provides the possibility to configure behavior, options, environment variables, and the logs files of Lisk Service with PM2.

Reviewer: ```suggestion
The {url_pm2_ecosystem_file}[PM2 ecosystem file^] provides the possibility to configure behavior, options, environment variables, and the logs files of Lisk Service with PM2.
```

Author: Can't really spot it - what is the difference here?

Reviewer: Sorry, I should have written the entire sentence..here it is:
The {url_pm2_ecosystem_file}[PM2 ecosystem file^] provides the possibility to configure behavior, options, environment variables, and the log files of Lisk Service with PM2.             (Not logs files)

Author: ah, I see, yes you are right :)

Author: ```suggestion
The {url_pm2_ecosystem_file}[PM2 ecosystem file^] provides the possibility to configure behavior, options, environment variables, and the log files of Lisk Service with PM2.
```

== The ecosystem config files

> The ecosystem file^]

Reviewer 2: Did you intend to remove this line?

Author: Yes you are right! Removed now :)

.Starts Lisk Service with the specified configuration
[source,bash]
----

----

=== Default configurations

The {url_pm2_ecosystem_file}[PM2 ecosystem file^] provides the possibility to configure behavior, options, environment variables, and the logs files of Lisk Service with PM2.

> The default confoguration file is `ecosystem.core3.config.js`, located in the root folder of `lisk-service`:

Reviewer 2: ```suggestion
The default configuration file is `ecosystem.core3.config.js`, located in the root directory of `lisk-service`:
```

It contains a configuration to connect to a local Lisk Core node.

This config will be used by default when starting Lisk Service with the following:

[source,bash]
----

> pm2 start
> 
> # or
> # pm2 start ecosystem.core3.config.js

Reviewer 2: Kindly consider updating it to the following as we do not have a default `ecosystem.config.js` file.
```suggestion
pm2 start ecosystem.core3.config.js
```
